core/editor.js

```diff
@@ -1648,6 +1648,83 @@
 
 		return element;
 	};
+
+	/**
+	 * Delay editor creation if given element is detached from DOM.
```

Comment:
> We mark it as private, but it is actually used as a public

Yeah, that's sometimes an issue that's not really private. But marking it with `@private` makes it more clear when reading API docs that one should not use this method.

> I'm wondering if `initializeDelayedEditorCreation` could be also split because we have there two exclusive logic

I would leave it as is.